index.js

```diff
@@ -1,3 +1,7 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
We normally add a copyright statement with the date to the top of the notice:
```
/* Copyright (c) 2022 The Brave Authors. All rights reserved.
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this file,
 * You can obtain one at https://mozilla.org/MPL/2.0/. */
```

Reply:
That's what I was curious! Sometimes I ran into copyrights without the year. Thank you for confirmation. Done.

Reply:
Indeed, there are a lot of old / incorrect ones still around in the codebase and so it's easy to forget to add it if copy/pasting from one of those files.